app/package.json

```diff
@@ -19,7 +19,7 @@
     "copy-plugins": "npx --no-install rimraf build/.plugins && mkdirp build/.plugins && copyfiles ../.plugins build/.plugins"
   },
   "build": {
-    "appId": "com.kinvolk.headlamp",
```

Comment:
Will this change affect upgrades?

Reply:
Hmm, good question. I don't think this being used anywhere, but we should double check.